File: Frontend/projects/edu-sharing-api/src/lib/api/models/admin-statistics.ts
/* tslint:disable */
/* eslint-disable */
import { Node } from './node';
export interface AdminStatistics {
    activeLocks?: Array<Node>;
    activeSessions?: number;
    allocatedMemory?: number;
    maxMemory?: number;
    numberOfPreviews?: number;
    previewCacheSize?: number;
}

File: Backend/services/core/src/main/java/org/edu_sharing/restservices/admin/v1/model/AdminStatistics.java
package org.edu_sharing.restservices.admin.v1.model;

;

import java.util.List;

import lombok.Data;
import org.edu_sharing.restservices.shared.Node;

@Data
public class AdminStatistics  {
  private int activeSessions,numberOfPreviews;
  private long maxMemory,allocatedMemory,previewCacheSize;
  private List<Node> activeLocks;
}
